test/builtin_dataset_mocks.py

```diff
@@ -1329,20 +1329,20 @@ def cub200(info, root, config):
 
 @register_mock
 def eurosat(info, root, config):
-    data_folder = pathlib.Path(root, "eurosat", "2750")
+    data_folder = root / "2750"
```

Comment:
How come the linux tests were passing before with the extra `eurosat` folder?
I would assume that if we change the structure in the , this would have to be reflected somehow in the dataset code?

Reply:
Since the prototype datasets never look at the absolute path, but only relative starting from the right

https://github.com/pytorch/vision/blob/27745e5be086ecdf2a8e609d65c13969cb5201f7/torchvision/prototype/datasets/_builtin/eurosat.py#L38

adding more folders on the left side makes no difference.